### PR TITLE
fix(docs-infra): Ensure experimental tag shows up on docs

### PR DESCRIPTION
**Pull request by atscott**
The experimental tag is part of the stability tags along with "deprecated". This commit updates some code to pick up experimental as well.

### CODE REVIEW

**Review by jelbourn — APPROVED** [review]
LGTM

**Review by jessicajaniuk — APPROVED** [review]
LGTM

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -183,6 +183,12 @@ This may be because we want to gather feedback from real applications before sta
 
 The policies and practices that are described in this document do not apply to APIs marked as Developer Preview. Such APIs can change at any time, even in new patch versions of the framework. Teams should decide for themselves whether the benefits of using Developer Preview APIs are worth the risk of breaking changes outside of our normal use of semantic versioning.
 
+## Experimental
+
+These are APIs might not become stable at all or have significant changes before becoming stable.
+
+The policies and practices that are described in this document do not apply to APIs marked as Developer Preview. Such APIs can change at any time, even in new patch versions of the framework. Teams should decide for themselves whether the benefits of using experimental APIs are worth the risk of breaking changes outside of our normal use of semantic versioning.
```
Comment (AndrewKushnir) [inline aio/content/guide/releases.md:190]: "Developer Preview" -> "experimental" in this paragraph.

**Review by AndrewKushnir — APPROVED** [review]
👍

**Comment by AndrewKushnir** [comment]
FYI, the `test` CI failure looks legit, adding a "cleanup" label.

**Pull request closed without merging**